zilencer/management/commands/populate_db.py

```diff
@@ -467,6 +469,25 @@ def handle(self, *args: Any, **options: Any) -> None:
 
             create_users(zulip_realm, names, tos_version=settings.TOS_VERSION)
 
+            users = UserProfile.objects.filter(realm=zulip_realm)
+            # All users in development environment are full members initially because
+            # waiting period threshold is 0. Groups of Iago, Dedemona, Shiva and
+            # Polonius will be updated according to their role in do_change_user_role.
+            full_members_user_group = UserGroup.objects.get(
+                realm=zulip_realm, name="@role:fullmembers", is_system_group=True
+            )
+            members_user_group = UserGroup.objects.get(
+                realm=zulip_realm, name="@role:members", is_system_group=True
+            )
+            user_group_memberships = []
+            for user_profile in list(users):
+                for group in [full_members_user_group, members_user_group]:
+                    user_group_membership = UserGroupMembership(
+                        user_group=group, user_profile=user_profile
+                    )
+                    user_group_memberships.append(user_group_membership)
+            UserGroupMembership.objects.bulk_create(user_group_memberships)
```

Comment:
This logic is a bit messy.  It's possible we should rework how these initial users are created to go through `do_create_user` with the role passed in, but I think that's out of scope for this PR.